src/App.js

```diff
@@ -51,15 +50,28 @@ function App() {
     setCart(newCart);
   };
 
-  const removeFromCart = (product) => {
-    const newCart = cart.map((item) => {
+  const decreaseQuantity = (product) => {
```

Comment:
it might be a good idea to combine this function and the next one like this:
const removeFromCart = (product, quantity)=>{
if (quantity) { remove this number from count}
else { remove this item totally from the cart}
}